src/test/java/bizbook/logic/commands/DeleteNotesCommandTest.java

```diff
@@ -0,0 +1,153 @@
+package bizbook.logic.commands;
+
+import static bizbook.logic.commands.CommandTestUtil.VALID_NOTE_HIGH_PROFILE_CLIENT;
+import static bizbook.testutil.Assert.assertThrows;
+import static bizbook.testutil.PersonBuilder.DEFAULT_ADDRESS;
+import static bizbook.testutil.PersonBuilder.DEFAULT_EMAIL;
+import static bizbook.testutil.PersonBuilder.DEFAULT_NAME;
+import static bizbook.testutil.PersonBuilder.DEFAULT_PHONE;
+import static bizbook.testutil.TypicalIndexes.INDEX_FIRST_NOTE;
+import static bizbook.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static bizbook.testutil.TypicalIndexes.INDEX_OUTOFBOUND_NOTE;
+import static bizbook.testutil.TypicalIndexes.INDEX_OUTOFBOUND_PERSON;
+import static bizbook.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+
+import bizbook.commons.core.index.Index;
+import bizbook.logic.Messages;
+import bizbook.logic.commands.exceptions.CommandException;
+import bizbook.model.Model;
+import bizbook.model.person.Person;
+import bizbook.testutil.PersonBuilder;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+public class DeleteNotesCommandTest {
+
+    @Test
+    public void constructor_nullFields_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new DeleteNotesCommand(null, null));
+    }
+
+    @Test
+    public void execute_validNoteDeletion_deletionSuccessful() throws Exception {
+
+        // Mock the model
+        Model modelMock = mock(Model.class);
+
+        // Create a valid person with a note and set up the model's person list
+        Person validPerson = new PersonBuilder().withNotes(VALID_NOTE_HIGH_PROFILE_CLIENT).build();
+        ObservableList<Person> personList = FXCollections.observableArrayList(validPerson);
+
+        // Mock the behavior of modelMock to return personList when needed
+        when(modelMock.getFilteredPersonList()).thenReturn(personList);
+
+        // Mock focused person property
+        ObjectProperty<Person> focusedPersonProperty = new SimpleObjectProperty<>(validPerson);
+        when(modelMock.getFocusedPerson()).thenReturn(focusedPersonProperty);
+
+        // Set up the person index and note index
+        Index validPersonIndex = INDEX_FIRST_PERSON;
+        Index validNoteIndex = Index.fromOneBased(1);
```

Comment:
I think this one you should add it as the `INDEX_FIRST_PERSON` as what was done in the line above for reusability.

So like the first person for the test address book will have 2 notes